packages/gatsby/src/cache/cache-fs.ts

```diff
@@ -54,7 +54,7 @@ exports.create = function (args): typeof DiskStore {
   return new DiskStore(args && args.options ? args.options : args)
 }
 
-function DiskStore(options): void {
+function DiskStore(this: any, options): void {
```

Comment:
Isn't this equal to DiskStore? (I'm also new to this in typescript :joy:)

Reply:
So this was about having it implicitly:

> TS2683: 'this' implicitly has type 'any' because it does not have a type annotation.

I didn't want to type everything so this is the best middleground for now